packages/miniflare/src/workers/kv/namespace.worker.ts

```diff
@@ -73,6 +76,46 @@ function secondsToMillis(seconds: number): number {
 	return seconds * 1000;
 }
 
+async function processKeyValue(
+	obj: KeyValueEntry<unknown> | null,
+	type: string = "text",
```

Comment:
what are the possible values for this type? Can this really be any string? If not perhaps a union type would be a better fit?

Reply:
text | json

Reply:
having read through the code once, I'm wondering if this should be smth like `type: "text" | "json"` 🤔